### PR TITLE
Update test script to get new access token on Che WS removal

### PR DESCRIPTION
**Pull request by maysunfaisal**
Signed-off-by: Maysun J Faisal <maysun.j.faisal@ibm.com>

Issue: #1239 

Screen Shot 2019-11-21 at 6 08 38 PM
Screen Shot 2019-11-21 at 6 08 48 PM

### CODE REVIEW

**Comment by maysunfaisal** [comment]
@ssh24 can you PTAL?

File: src/pfe/file-watcher/server/test/scripts/setup.sh [hunk 1]
```diff
@@ -6,6 +6,13 @@ RED='\033[0;31m'
 BLUE='\033[0;36m'
 RESET='\033[0m'
 
+CHE_USER="admin"
+CHE_PASS="admin"
+
+KEYCLOAK_HOSTNAME=keycloak-"$NAMESPACE"."$CLUSTER_IP".nip.io
+TOKEN_ENDPOINT="http://${KEYCLOAK_HOSTNAME}/auth/realms/che/protocol/openid-connect/token"
+CHE_ACCESS_TOKEN=$(curl -sSL --data "grant_type=password&client_id=che-public&username=${CHE_USER}&password=${CHE_PASS}" ${TOKEN_ENDPOINT} | jq -r '.access_token')
```
Comment (ssh24) [inline src/pfe/file-watcher/server/test/scripts/setup.sh]: I think we do it like this when generating the first time as well. I am wondering if we can make it a global function on the init script and call it every time we want regenerate one?

Comment (maysunfaisal) [reply src/pfe/file-watcher/server/test/scripts/setup.sh]: will do

Comment (maysunfaisal) [reply src/pfe/file-watcher/server/test/scripts/setup.sh]: @ssh24 I've updated the PR, can you PTAL

**Comment by maysunfaisal** [comment]
@rajivnathan can you pls help merge this for me? thx

**Pull request merged**